Dockerfile

```
@@ -2,20 +2,20 @@
 FROM node:20.5.1-alpine as base
 WORKDIR /home/node/app
 
-# Build layer (everything besides build output & node_modules can be discarded from this)
+# Build layer
 FROM base as build
 
 COPY yarn.lock package.json ./
-RUN yarn --production
+RUN yarn install --frozen-lockfile
 COPY . .
 RUN yarn run build
 
-# Production layer, only copy required files to run.
+# Production layer
 FROM base as production
 
 ENV NODE_ENV=production
 COPY --from=build /home/node/app/package.json ./
-COPY --from=build /home/node/app/node_modules ./node_modules
 COPY --from=build /home/node/app/dist ./dist
+RUN yarn install --frozen-lockfile --production
```

this would reinstall all packages for every changed line, try this:

```suggestion
ENV NODE_ENV=production
COPY yarn.lock package.json ./
RUN yarn install --frozen-lockfile --production
COPY --from=build /home/node/app/dist ./dist
```

---

okidoki done, github marked as outdated suggestion so couldn't autoapply :angery: